src/media/interface-iwpml-pb-media-node-iterator.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+interface IWPML_PB_Media_Node_Iterator {
```

Comment:
@strategio I don't think `_Iterator` makes sense here. The doesn't not iterate over anything that I can see. Isn't it just a node?

Reply:
@brucepearson, actually it should be `IWPML_PB_Media_Nodes_Iterator` (with a "s" for Nodes) because traverse all the media nodes and try to translate it.